components/kyma-environment-broker/internal/hyperscaler/azure/config.go

```diff
@@ -43,3 +43,75 @@ func (c *Config) Environment() (*azure.Environment, error) {
 
 	return c.environment, nil
 }
+
+func azureRegions() map[string]bool {
```

Comment:
use ` map[string]struct{}` instead of `map[string]bool`.

`struct` [occupies zero bytes of storage](https://dave.cheney.net/2014/03/25/the-empty-struct) (here is no need to have a bool flag).

Reply:
the bool makes the check for membership really nice to read. But if size is of concern here i will change it

Reply:
done